app/user/user.js

```diff
@@ -81,6 +81,22 @@ User.prototype.removeInviteInst = function removeInviteInst(institutionKey) {
     });
 };
 
+User.prototype.updateInstitutionAndFollow = function updateInstitutionAndFollow(institution){
```

Comment:
se tem um And no nome do método é pq ele está fazendo coisa demais, entretanto se o follow é efeito colateral ele não precisa vir no nome.